tests.yaml

```diff
@@ -823,26 +823,25 @@ tests:
         description: "The apiserver has terminated 33.33% of its incoming requests."
         runbook_url: "https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-kubeapiterminatedrequests"
 
-- interval: 30s
+- interval: 1m
   input_series:
-  - series: 'kube_pod_container_status_restarts_total{namespace="test",pod="static-web",container="script",job="kube-state-metrics"}'
-    values: '0 1 2 3 4+0x3 5+0x6 6+0x12 7+0x12 8+0x33'
-  - series: 'kube_pod_container_status_waiting{namespace="test",pod="static-web",container="script",job="kube-state-metrics"}'
-    values: '1+0x38 0+0x32'
+  - series: 'kube_pod_container_status_waiting_reason{reason="CrashLoopBackOff",namespace="test",pod="static-web",container="script",job="kube-state-metrics"}'
+    values: '1 1 0+0x4 1 1 0+0x3 1 1 0+0x20'
```

Comment:
can you add another test with `stale` values since this is closer to what kube-state-metrics exposes?

Reply:
I'll just replace the trailing zeros with stale then actually since the if the metric is stale there is no point in testing a not-stale state, correcting the test will highlight/document the expected behaviour